eng/ci/custom-image-tests.yml

```diff
@@ -0,0 +1,27 @@
+resources:
```

Comment:
Yep, it will be a manual test workflow, like we do currently. In the pipeline, you'll be able to set the DEDICATED_DOCKER_TEST, CONSUMPTION_DOCKER_TEST, and IMAGE_NAME variables.